src/utils/constants/constants.js

```diff
@@ -1,4 +1,3 @@
-export const constants = {
-  DEFUALT_CRITERIA: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-}
+export const DEFAULT_CRITERIA = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
Good stuff. I would maybe take this one step further and have constant values for each status. I.e.

`
export const STATUS = {
  EMPTY: 0,
  RED: 1,
  AMBER: 2,
  GREEN: 3
}
`
I feel like this would give a bit more meaning to `DEFAULT_CRITERIA` other than it just being an array of zero's. Instead it would be [STATUS.EMPTY, STATUS.EMPTY...]

Reply:
I like it - although I would suggest that we only use the STATUS.EMPTY in the constants file (where we create the list of 0's) and then the others in the TeamStatus.vue file.

Reply:
I agree with you there. If you can think of a better name for the empty status, then go ahead. Maybe `STATUS.NOT_SET` is a bit more informative